examples/bottom_navigation_multiple_beamers/lib/main.dart

```diff
@@ -164,94 +156,113 @@ class ArticlesLocation extends BeamLocation {
   List<BeamPage> pagesBuilder(BuildContext context, BeamState state) => [
         BeamPage(
           key: ValueKey('articles'),
+          title: 'Articles',
+          type: BeamPageType.noTransition,
           child: ArticlesScreen(),
         ),
         if (state.pathParameters.containsKey('articleId'))
           BeamPage(
             key: ValueKey('articles-${state.pathParameters['articleId']}'),
+            title: articles.firstWhere((article) =>
+                article['id'] == state.pathParameters['articleId'])['title'],
             child: ArticleDetailsScreen(
-              articleId: state.pathParameters['articleId'],
+              article: articles.firstWhere((article) =>
+                  article['id'] == state.pathParameters['articleId']),
             ),
           ),
       ];
 }
 
 // APP
 class AppScreen extends StatefulWidget {
-  AppScreen({this.beamState});
-
+  AppScreen({required this.beamState});
   final BeamState beamState;
 
   @override
   _AppScreenState createState() => _AppScreenState();
 }
 
 class _AppScreenState extends State<AppScreen> {
-  final _routerDelegates = [
+  late int currentIndex;
+
+  final routerDelegates = [
     BeamerRouterDelegate(
-      locationBuilder: (state) => ArticlesLocation(state),
+      locationBuilder: (state) => BooksLocation(state),
     ),
     BeamerRouterDelegate(
-      locationBuilder: (state) => BooksLocation(state),
+      locationBuilder: (state) => ArticlesLocation(state),
     ),
   ];
 
-  int _currentIndex;
-
   @override
   void initState() {
+    currentIndex = widget.beamState.uri.path.contains('books') ? 0 : 1;
     super.initState();
```

Comment:
call to super must be first.

https://api.flutter.dev/flutter/widgets/State/initState.html:
"If you override this, make sure your method starts with a call to super.initState()."